src/main/java/me/botsko/prism/listeners/PrismPlayerEvents.java

```diff
@@ -324,9 +324,15 @@ public void onPlayerInteract(final PlayerInteractEvent event) {
             // " Item in hand:" + player.getItemInHand().getTypeId() + ":" +
             // player.getItemInHand().getDurability());
 
+            int itemInHandId = player.getItemInHand().getTypeId();
```

Comment:
Not sure you understood my comment. I wasn't asking you to remove a cache variable, I was asking you to document, in code comments, why this new code block is necessary.